src/main/resources/application.yaml

```diff
@@ -114,3 +114,9 @@ rhsm-subscriptions:
   prometheus-latency-duration: ${PROMETHEUS_LATENCY_DURATION:4h}
   hourly-tally-offset: ${HOURLY_TALLY_OFFSET:60m}
   metric-lookup-range-duration: ${METRIC_LOOKUP_RANGE:24h}
+  subscription:
+    useStub: ${SUBSCRIPTION_USE_STUB:true}
+    url: ${SUBSCRIPTION_URL:https://subscription.qa.api.redhat.com/svcrest/subscription/v5}
+    keystore: file:${SUBSCRIPTION_KEYSTORE:}
+    keystorePassword: ${SUBSCRIPTION_KEYSTORE_PASSWORD:changeit}
+    maxConnections: ${SUBSCRIPTION_MAX_CONNECTIONS:100}
```

Comment:
Can you please populate these env vars in `templates/marketplace-worker.yaml`. They should replace `MARKETPLACE_DUMMY_ID`?

For populating `SUBSCRIPTION_KEYSTORE` and `SUBSCRIPTION_KEYSTORE_PASSWORD`, `templates/rhsm-conduit.yaml` -> `RHSM_KEYSTORE`, etc. is a good reference (we will use the same secret).